dashboard-ui/src/pages/home.tsx

```diff
@@ -33,13 +33,19 @@ import type { SortBy } from '@kubetail/ui/elements/DataTable/Header';
 import Form from '@kubetail/ui/elements/Form';
 import Spinner from '@kubetail/ui/elements/Spinner';
 
+import FormControl from '@kubetail/ui/elements/FormControl';
```

Comment:
Let's group this import with `Form` and `Spinner` above. In general we use four categories of imports grouped separately at the top of every page: standard library, third-party, kubetail-external, internal